crates/oxc_ast/src/ast/js.rs

```diff
@@ -101,49 +102,88 @@ inherit_variants! {
 #[cfg_attr(feature = "serialize", derive(Serialize, Tsify))]
 #[cfg_attr(feature = "serialize", serde(untagged))]
 pub enum Expression<'a> {
+    /// See [`BooleanLiteral`] for ast node details.
```

Comment:
This is a real nit, but could we please use capitalized "AST" rather than "ast"? This isn't consistent throughout Oxc's codebase as present, but since you're going to the trouble of documenting the AST, it'd be great to set a good pattern here.